### PR TITLE
Added stationname as label

### PR DESCRIPTION
**Pull request by andreabl**

### CODE REVIEW

File: collector.go [hunk 1]
```diff
@@ -122,43 +124,45 @@ func collectData(ch chan<- prometheus.Metric, device *netatmo.Device) {
...
 	if data.Rain != nil {
-		sendMetric(ch, rainDesc, prometheus.GaugeValue, float64(*data.Rain), moduleName)
+		sendMetric(ch, rainDesc, prometheus.GaugeValue, float64(*data.Rain), moduleName, stationName)
 	}
 }
 
-func sendMetric(ch chan<- prometheus.Metric, desc *prometheus.Desc, valueType prometheus.ValueType, value float64, moduleName string) {
-	m, err := prometheus.NewConstMetric(desc, valueType, value, moduleName)
+func sendMetric(ch chan<- prometheus.Metric, desc *prometheus.Desc, valueType prometheus.ValueType, value float64, moduleName string, stationName string) {
+
```
Comment (xperimental) [inline collector.go]: Could you remove this line?

File: collector.go [hunk 2]
```diff
@@ -122,43 +124,45 @@ func collectData(ch chan<- prometheus.Metric, device *netatmo.Device) {
...
+	m, err := prometheus.NewConstMetric(desc, valueType, value, moduleName, stationName)
+
```
Comment (xperimental) [inline collector.go]: And this one?

**Comment by xperimental** [comment]
Another question occurred to me today: Do you own a wind and/or rain sensor? The code for those two sensors was also contributed but I do not own those sensors and so could not test it, so if you have one or two of the sensors I would appreciate it if you could tell me if those are working properly.

**Comment by andreabl** [comment]
Hi @xperimental and thanks for the feedback.

I have tried to update the PR with my proper address and with the two newlines removed. Does it work?

Unfortunately, I do not have access to rain and/or wind sensors here. I guess it should be just fine, but it would be nice to have it tested. 

Another potential issue of my patch is that Prometheus will treat the metrics with the stationname-label as new metrics, which might or might not be a problem, depending on your setup.

**Comment by xperimental** [comment]
That's unfortunate that you also do not have a wind or rain sensor. Well, let's do a release anyway, it has been a while already...

Thanks for the note about the changed metric name. My existing queries should still work properly.

**Pull request merged**